src/LoginView.js

```diff
@@ -113,13 +114,15 @@ export default class LoginView extends PureComponent {
         currentUri.indexOf(adService.passwordResetPolicy) > -1
           ? adService.passwordResetPolicy
           : adService.loginPolicy;
+      this.setState({ renderExitDone: true, renderExit: false });
```

Comment:
Can you just explain to me why `renderExitDone=true` when `result.requestType === RequestType.Code` ?

Reply:
Code == possible new token. If the token is accepted `onSuccess()` will be triggered (see line 123) and the exit button should be disabled. Had to put this here since to update the state and rerender. If the request fails we set this back (line 125) to make the button visible in the next component update

Reply:
And when the user clicks `TIlbake til hjem`, then the whole `LoginView` is closed. Therefore `renderExitDone` does not have to be set in the `Cancelled` case?

Reply:
Yes. "Cancelled" call in this case is when hitting the "Cancel"-button to return to the main login view. 

To give you some context the `renderExitDone` is an additional variable to remove the exit button at the redirect page. 

There are two situations where we want to disable the button:
1. During loading, when the `WebView` is in loading mode, and
2. At the redirect screen when we wait for the `WebView` to close after a successful login.

Roughly, `renderExit` handles the first case alone, but `renderExitDone` is needed in addition to stop the button rendering AFTER a successful login to handles the second case. This is because the `WebView` exits the loading state after redirect. So `renderExitDone` basically means "don't enable the button after loading the next web page"